scripts/test-run-nats-embedded

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+start-test() {
+    local ip=$(cat $TEST_DIR/databases/*/metadata/ip)
+    local port=$(cat $TEST_DIR/databases/*/metadata/port)
+    KINE_IMAGE=$IMAGE KINE_ENDPOINT="nats://$ip:$port?embedServer" provision-kine
```

Comment:
In the case of an embedded server there's no dedicated database container, right? So the IP and port shouldn't be set.
```suggestion
    KINE_IMAGE=$IMAGE KINE_ENDPOINT="nats://?embedServer" provision-kine
```